plugins/PrivacyManager/changes.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "title":       "Privacy Manager - User Opt-Out Improvements",
+    "description": "The privacy manager user opt-out has been updated to provide new code generation options that improve compatibility and allow additional customisation.",
+    "version":     "4.12.0"
```

Comment:
If we plan to write some kind of documentation around the new opt out, we could include the link here.